Create product-docs-may.md (#3630)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/08-May-2023/product-docs-may.md

---
title: "May 2023 product docs updates"
id: "May-product-docs"
description: "May 2023: Find out what the product docs team has been busy doing in the month of May."
sidebar_label: "Update: product docs changes"
sidebar_position: 1
tags: [May-2023,product-docs]

> ---

Reviewer: ```suggestion
date: 06-01-2023
---
```

## ☁ Cloud projects

- We launched [dbt Cloud IDE user interface doc](/docs/cloud/dbt-cloud-ide/ide-user-interface), which provides a thorough walkthrough of the IDE UI elements and their definitions.
- Launched a sparkling new [dbt Cloud Scheduler page](/docs/deploy/job-scheduler) ✨! We went from previously having little content around the scheduler to a subsection that breaks down the awesome scheduler features and how it works.
- Updated the [dbt Cloud user license page](/docs/cloud/manage-access/seats-and-users#licenses) to clarify how to add or remove cloud users.
- Shipped these Discovery API docs to coincide with the launch of the Discovery API:
    - [About the Discovery API](/docs/dbt-cloud-apis/discovery-api)
    - [Use cases and examples for the Discovery API](/docs/dbt-cloud-apis/discovery-use-cases-and-examples)
    - [Query the Discovery API](/docs/dbt-cloud-apis/discovery-querying)

## 🎯 Core projects

- See what’s coming up [in Core v 1.6](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.6%22)!

> - We turned the `profiles.yml` [page](/reference/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.

Author: ```suggestion
- We turned the `profiles.yml` [page](/docs/core/connect-data-platform/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.
```

---
title: "May 2023 product docs updates"
id: "May-product-docs"
description: "May 2023: Find out what the product docs team has been busy doing in the month of May."

> sidebar_label: "Update: product docs changes"

Reviewer: ```suggestion
sidebar_label: "Update: Product docs changes"
```

> sidebar_position: 8

Author: ```suggestion
sidebar_position: 1
```

> tags: [May-2023,product-docs]

Reviewer: ```suggestion
tags: [May-2023, product-docs]
```

> date: 06-01-2023

Author: ```suggestion
date: 2023-06-01
```

---

## May **2023 product docs updates**

Hello from the dbt Docs team: @mirnawong1, @matthewshaver, @nghi-ly, and @runleonarun! First, we’d like to thank the 13 new community contributors to docs.getdbt.com!

> What's new to [docs.getdbt.com](http://docs.getdbt.com/) in May:

Author: ```suggestion
Here's what's new to [docs.getdbt.com](http://docs.getdbt.com/) in May:
```

- Added a new [Quickstarts landing page](/quickstarts). This new format sets up for future iterations that will include filtering! But for now, we are excited you can step through quickstarts in a focused way.

## ☁ Cloud projects

- We launched [dbt Cloud IDE user interface doc](/docs/cloud/dbt-cloud-ide/ide-user-interface), which provides a thorough walkthrough of the IDE UI elements and their definitions.
- Launched a sparkling new [dbt Cloud Scheduler page](/docs/deploy/job-scheduler) ✨! We went from previously having little content around the scheduler to a subsection that breaks down the awesome scheduler features and how it works.
- Updated the [dbt Cloud user license page](/docs/cloud/manage-access/seats-and-users#licenses) to clarify how to add or remove cloud users.
- Shipped these Discovery API docs to coincide with the launch of the Discovery API:
    - [About the Discovery API](/docs/dbt-cloud-apis/discovery-api)
    - [Use cases and examples for the Discovery API](/docs/dbt-cloud-apis/discovery-use-cases-and-examples)
    - [Query the Discovery API](/docs/dbt-cloud-apis/discovery-querying)

>     - [Project state in dbt Cloud](/docs/deploy/project-state#project-state-in-dbt-cloud)

Author: ```suggestion
```

## 🎯 Core projects

- See what’s coming up [in Core v 1.6](https://github.com/dbt-labs/docs.getdbt.com/issues?q=is%3Aissue+label%3A%22dbt-core+v1.6%22)!
- We turned the `profiles.yml` [page](/docs/core/connect-data-platform/profiles.yml) into a landing page, added more context to profile.yml page, and moved the ‘About CLI’ higher up in the `Set up dbt` section.

## New 📚 Guides, ✏️ blog posts, and FAQs

If you want to contribute to a blog post, we’re focusing on content

> - **[Accelerate your documentation workflow: Generate docs for whole folders at once](/blog/generating-dynamic-docs-dbt)**

Author: ```suggestion
- Published a blog post: [Accelerate your documentation workflow: Generate docs for whole folders at once](/blog/generating-dynamic-docs-dbt)
```

> - **[Data engineers + dbt v1.5: Evolving the craft for scale](/blog/evolving-data-engineer-craft)**

Author: ```suggestion
- Published a blog post: [Data engineers + dbt v1.5: Evolving the craft for scale](/blog/evolving-data-engineer-craft)
```

> - Added an [FAQ](/faqs/Warehouse/db-connection-dbt-compile) to clarify the common question users have of *Why does dbt compile need to connect to the database?*

Reviewer: ```suggestion
- Added an [FAQ](/faqs/Warehouse/db-connection-dbt-compile) to clarify the common question users have on *Why does dbt compile needs to connect to the database?*
```

> - Published a [discourse article](https://discourse.getdbt.com/t/how-to-configure-external-user-email-notifications-in-dbt-cloud/8393) about configure job notifications for non-dbt Cloud users

Reviewer: ```suggestion
- Published a [discourse article](https://discourse.getdbt.com/t/how-to-configure-external-user-email-notifications-in-dbt-cloud/8393) about configuring job notifications for non-dbt Cloud users
```
